modules/issue_tracker/php/issuerow.class.inc

```diff
@@ -29,18 +29,23 @@ class IssueRow implements \LORIS\Data\DataInstance
 {
     protected $DBRow;
     protected $CenterID;
+    protected $Module;
 
     /**
      * Create a new IssueRow
      *
-     * @param array   $row The row (in the same format as \Database::pselectRow
-     *                     returns
-     * @param integer $cid The centerID affiliated with this row.
+     * @param array   $row    The row (in the same format as \Database::pselectRow
+     *                        returns.
+     * @param integer $cid    The centerID affiliated with this row.
+     * @param \Module $module The module that this issue is for.
```

Comment:
This is a cool feature. It would be good to update the TestPlan to provide some instructions to make sure it's working.

Reply:
As far as I know this doesn't change the feature other than using the abstract function to get the name